### PR TITLE
fix: change log

### PR DESCRIPTION
**Pull request by yuanganping**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -44,16 +44,16 @@
...
 - Fixed the run command
-- Fixed stateful workload get config and state of ⚙️
+- Fixed stateful workload get config and state of ⚙️ icon
 
 ### Optimization
 
 - Improve the speed of opening application
-- Add prommpt wnen Open Project is not associated with a target
-- Unified text description information
+- Add prompt when Open Project is not associated with a target
```
Comment (xinxinh2020) [inline CHANGELOG.md:58]: Add prompt when Open Project is not associated with a target directory

File: CHANGELOG.md [hunk 2]
```diff
@@ -44,16 +44,16 @@
...
+- Unify menu and page text description
```
Comment (xinxinh2020) [inline CHANGELOG.md:59]: Unify description of menu and page

File: CHANGELOG.md [hunk 3]
```diff
@@ -44,16 +44,16 @@
...
 - Add status for cluster
-- No cluster add guide information
+- Add guide information when there is no cluster
 
 ### Bug Fixes
 
 - Fixed the ssh trust problem when installing application for the first time
 - Fixed the run command
-- Fixed stateful workload get config and state of ⚙️
+- Fixed stateful workload get config and state of ⚙️ icon
```
Comment (xinxinh2020) [inline CHANGELOG.md:53]: Fixed failing to get config of statefulset 
Fixed status icon of statefulset displaying incorrectly

File: CHANGELOG.md [hunk 4]
```diff
@@ -44,16 +44,16 @@
 
 - Add icon for syncService
 - Add status for cluster
-- No cluster add guide information
+- Add guide information when there is no cluster
 
 ### Bug Fixes
 
 - Fixed the ssh trust problem when installing application for the first time
 - Fixed the run command
```
Comment (xinxinh2020) [inline CHANGELOG.md:52]: Fixed `run` command using incorrect `nhctl` binary

**Review by anurnomeru — APPROVED** [review]
LGTM

**Pull request merged**